### PR TITLE
feat(core): added examples for eth v1 

### PR DESCRIPTION
**Pull request by AbhayShekhawat94**

### CODE REVIEW

**Review by argjv — APPROVED** [review]

File: modules/core/example/js/eth/create-wallet-address.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ // set your wallet from the YYYYY parameter here in the URL on app.bitgo-test.com
+ // https://test.bitgo.com/enterprise/XXXXXXXXX/coin/teth/YYYYY/transactions
+ const walletId = '';
+
+ // ETH only parameter -
+ // Use 0 for the old forwarder (https://github.com/BitGo/eth-multisig-v2),
+ // and 1 for the new fee-improved forwarder (https://github.com/BitGo/eth-multisig-v4).
+ const forwarderVersion = 1;
+
+ Promise.coroutine(function *() {
```
Comment (ryan-bitgo) [inline modules/core/example/js/eth/create-wallet-address.js:26]: for new code we should use asnyc/await

File: modules/core/example/js/eth/create-wallet-address.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ const BitGoJS = require('bitgo');
+ const Promise = require('bluebird');
+
+ // change this to env: 'production' when you are ready for production
+ const bitgo = new BitGoJS.BitGo({ env: 'test' });
+
+ // Change coin to 'eth' when working with production
+ const coin = 'teth';
+ // TODO: set your access token here
+ const accessToken = '';
```
Comment (ryan-bitgo) [inline modules/core/example/js/eth/create-wallet-address.js:15]: environment variable can work here as well

File: modules/core/example/js/eth/create-wallet-address.js (hunk 1 above)
Comment (AbhayShekhawat94) [reply modules/core/example/js/eth/create-wallet-address.js]: Done!

File: modules/core/example/js/eth/create-wallet-address.js (hunk 2 above)
Comment (AbhayShekhawat94) [reply modules/core/example/js/eth/create-wallet-address.js:14]: All the other examples follow the same pattern, so to keep it consistent throughout, I just kept it like this

**Review by barathcj — APPROVED** [review]

File: modules/core/example/js/eth/create-wallet-address.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+/**
+ * Create a new receive address on an existing multi-sig ethereum wallet at BitGo.
+ *
+ * Copyright 2021, BitGo, Inc.  All Rights Reserved.
+ */
+ const BitGoJS = require('bitgo');
+
+ // change this to env: 'production' when you are ready for production
+ const bitgo = new BitGoJS.BitGo({ env: 'test' });
```
Comment (tylerlevine) [inline modules/core/example/js/eth/create-wallet-address.js:9]: Nit: could provide the access token using the `accessToken` constructor param.

File: modules/core/example/js/eth/create-wallet-address.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ // https://test.bitgo.com/enterprise/XXXXXXXXX/coin/teth/YYYYY/transactions
+ const walletId = '';
+
+ // ETH only parameter -
+ // Use 0 for the old forwarder (https://github.com/BitGo/eth-multisig-v2),
+ // and 1 for the new fee-improved forwarder (https://github.com/BitGo/eth-multisig-v4).
+ const forwarderVersion = 1;
+
+ async function createAddress() {
+   bitgo.authenticateWithAccessToken({ accessToken: accessToken });
```
Comment (tylerlevine) [inline modules/core/example/js/eth/create-wallet-address.js:26]: nit
```suggestion
   bitgo.authenticateWithAccessToken({ accessToken });
```

File: modules/core/example/js/eth/create-wallet-address.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+   const wallet = await bitgo.coin(coin).wallets().get({ id: walletId });
+
+   const newReceiveAddress = await wallet.createAddress({ forwarderVersion: forwarderVersion });
```
Comment (tylerlevine) [inline modules/core/example/js/eth/create-wallet-address.js:30]: nit
```suggestion
   const newReceiveAddress = await wallet.createAddress({ forwarderVersion });
```

File: modules/core/example/js/eth/start-stop-manual-deployment.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+async function startOrStopManualDeployment() {
+  bitgo.authenticateWithAccessToken({ accessToken });
+
+  // set to 'true' to start manual deployment and 'false' to stop manual deployment
+  const deployForwardersManually = true;
+  // set to 'true' to start manual flushing and 'false' to stop manual flushing
+  const flushForwardersManually = true;
+
+  // Refer https://app.bitgo.com/docs/#operation/v2.wallet.get for all different payload options
+  const ethSpecificPayload = { [coin]: { deployForwardersManually: deployForwardersManually, flushForwardersManually: flushForwardersManually } } ;
```
Comment (tylerlevine) [inline modules/core/example/js/eth/start-stop-manual-deployment.js:32]: nit
```suggestion
  const ethSpecificPayload = { [coin]: { deployForwardersManually, flushForwardersManually } } ;
```

File: modules/core/example/js/eth/start-stop-manual-deployment.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+  const body = { approvalsRequired: 1, coinSpecific: ethSpecificPayload };
+
+  const response = await bitgo.put(url).send(body).result();
```
Comment (tylerlevine) [inline modules/core/example/js/eth/start-stop-manual-deployment.js:35]: Why don't we have a SDK method for this? We should not encourage usage of raw HTTP methods from callers - these methods should only be used internally.

**Review by tylerlevine — CHANGES_REQUESTED** [review]
only nits, except we should also have typescript examples please (we should not encourage further javascript usage by SDK consumers).

File: modules/core/example/js/eth/create-wallet-address.js (hunk 3 above)
Comment (AbhayShekhawat94) [reply modules/core/example/js/eth/create-wallet-address.js:9]: added typescript examples

File: modules/core/example/js/eth/start-stop-manual-deployment.js (hunk 7 above)
Comment (AbhayShekhawat94) [reply modules/core/example/js/eth/start-stop-manual-deployment.js]: Moved  example to express integration tests

**Review by tylerlevine — APPROVED** [review]

**Pull request merged**